examples/clientcerts/clientcerts.values.yaml

```diff
@@ -0,0 +1,28 @@
+# These sections are used for the client side of certificate authentication.
+coderd:
+  clientTls:
+    certSecret:
+      name: "client-cert-secret"
+      key: "client-cert-filename.crt"
+    keySecret:
+      name: "client-key-secret"
+      key: "client-key-filename.key"
+
+postgres:
+  default:
+    enable: false
+  host: "some.rds.hostname"
+  port: 5432
+  user: "coder"
+  database: "coder"
+  sslMode: "require"
+  ssl:
+    certSecret:
+      name: "database-client-cert-secret"
+      key: "database-client-cert-filename.crt"
+    keySecret:
+      name: "database-client-key-secret"
+      key: "database-client-key-filename.key"
+    rootCertSecret:
+      name: "database-root-cert-secret"
+      key: "database-root-cert-filename.crt"
```

Comment:
needs a blank line at end of file

Reply:
updated examples file with the new values.